scripts/checkout_web_vault.sh

```diff
@@ -53,4 +53,7 @@ pushd "${VAULT_FOLDER}"
     git fetch --tags --depth 1 origin "${VAULT_VERSION}"
     # Checkout the branch we want
     git -c advice.detachedHead=false checkout FETCH_HEAD
+    # Make sure we don't use non-OSI code
+    # requires patching in next step
+    rm -rf bitwarden_license
```

Comment:
This should probably be done in the `apply_patches.sh` script so it will also be removed from the Docker build.

Also if you remove this directory you should also exclude it from the generated patch file:
https://github.com/dani-garcia/bw_web_builds/blob/cc03d26ac0f47481067f6ae313375e198a7b1164/scripts/generate_patch_file.sh#L40-L41

Reply:
I think it's too much for something not used at all.